### PR TITLE
Feat/tests

### PR DESCRIPTION
**Pull request by BelfordZ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,7 @@
-# OpenRPC Specification v1.0.0 JSON Schema
+# OpenRPC Specification 1.0.0-rc0 JSON Schema
```
Comment (BelfordZ) [inline README.md]: fix dis

**Comment by shanejonas** [comment]
these tests should be failing until these fixes get in:

https://github.com/open-rpc/examples/pull/11
https://github.com/open-rpc/meta-schema/issues/31

**Comment by BelfordZ** [comment]
They are not fully complete. Need to rework this so we can get coverage of generated client coverage as well.

**Comment by BelfordZ** [comment]
@shanejonas this PR will create #32

**Pull request merged**